### ISSUE TITLE
Support rust_name/cxx_name on enum variants

### ISSUE THREAD

**Issue by dtolnay** [dtolnay/cxx#379]
Attributes were added for renaming *functions* in #349. It would make sense to support the same on enum variants.

```rust
#[cxx::bridge]
mod ffi {
    enum Enum {
        #[cxx_name = "VARIANT"]
        Variant,
    }
}
```

**Comment by alexeyr** [comment]
What about structs/enums themselves (doesn't seem to work at the moment)? Or should this be a different issue?

**Comment by dtolnay** [comment]
Implemented in #588.
